locales/en/app.ftl

```diff
@@ -9,6 +9,7 @@
 -brand-name = Firefox
 -brand-Mozilla = Mozilla
 -brand-premium = Premium
+-brand-monitor-premium = Monitor Premium
```

Comment:
As mentioned [here](https://github.com/mozilla/blurts-server/pull/3022#discussion_r1187098252), we should define the term later in the file.

We will also need to add comments, once the branding has been clarified by product.